pyiron_workflow/composite.py

```diff
@@ -763,3 +763,7 @@ def _restore_signal_connections_from_strings(
             self._get_signals_input,
             self._get_signals_output,
         )
+
+    @property
+    def import_ready(self) -> bool:
+        return super().import_ready and all(node.import_ready for node in self)
```

Comment:
Most of the time, if the user hits a `TypeNotFound` error, I expect them to have an "aha" moment and know immediately which of their nodes is causing trouble. In case they _don't_, they should be able to do something like `for child in my_composite: print(child.label, child.import_ready)`, so it's nice to have as a fallback.

I have a related question though, which is whether it's worth supplying something like `Node.import_readiness_report` analogous to the existing `Node.readiness_report` that basically provides a recursive version of the snippet I gave above. Yesterday I leaned towards "no", but this morning I'm having second thoughts.